ci/install_travis.sh

```diff
@@ -90,55 +80,10 @@ echo
 echo "[create env]"
 
 # create our environment
-REQ="ci/requirements-${JOB}.build"
-time conda create -n pandas --file=${REQ} || exit 1
+time conda env create -q -n pandas --file="${ENV_FILE}" || exit 1
 
 source activate pandas
 
-# may have addtl installation instructions for this build
-echo
-echo "[build addtl installs]"
-REQ="ci/requirements-${JOB}.build.sh"
-if [ -e ${REQ} ]; then
-    time bash $REQ || exit 1
-fi
-
-time conda install -n pandas pytest>=3.1.0
-time pip install -q pytest-xdist moto
-
-if [ "$LINT" ]; then
-   conda install flake8=3.4.1
-   pip install cpplint
-fi
-
-if [ "$COVERAGE" ]; then
```

Comment:
where did coverage go?

Reply:
The coverage deps are run on the 3.6 build. `travis-36.yaml` has the additional coverage deps.